core/src/al/audio_context.cpp

```diff
@@ -12,3 +12,12 @@ std::shared_ptr<AudioContext> AudioContext::create()
     return nullptr;
 #endif
 }
+
+int AudioContext::getMaxListenerCount()
+{
+#ifdef CUBOS_CORE_MINIAUDIO
+    return MA_ENGINE_MAX_LISTENERS;
+#else
+    return 0;
+#endif
+}
```

Comment:
You are breaking the abstraction layer here. This file should know nothing about miniaudio.
Make the method virtual and move this to MiniaudioContext